kotlin-codepoints-deluxe/src/commonMain/kotlin/CodePointSequence.kt

```diff
@@ -0,0 +1,27 @@
+package de.cketti.codepoints.deluxe
+
+/**
+ * Sequence of [CodePoint]s in the given [String].
+ */
+class CodePointSequence(private val text: String) : Sequence<CodePoint> {
```

Comment:
```suggestion
@JvmInline
value class CodePointSequence(private val text: String) : Sequence<CodePoint> {
```
May as well try to avoid the allocation

Reply:
Done.